ci/jenkins/test-mc.sh

```diff
@@ -397,17 +397,18 @@ function run_multicluster_e2e {
     docker pull "${DOCKER_REGISTRY}"/antrea/nginx:1.21.6-alpine
     docker save "${DOCKER_REGISTRY}"/antrea/nginx:1.21.6-alpine -o "${WORKDIR}"/nginx.tar
 
-    docker pull "${DOCKER_REGISTRY}/antrea/agnhost:2.26"
-    docker tag "${DOCKER_REGISTRY}/antrea/agnhost:2.26" "agnhost:2.26"
-    docker save agnhost:2.26 -o "${WORKDIR}"/agnhost.tar
+    # Use the same agnhost image which is defined as 'agnhostImage' in antrea/test/e2e/framework.go to
+    # avoid pulling this image again when Multi-cluster e2e is running.
```

Comment:
"when running Multi-cluster e2e tests"

Reply:
"antrea/agnhost:2.26" is a different image from "registry.k8s.io/e2e-test-images/agnhost:2.29" right? Then change "this image" to "the image".

Reply:
Updated, thanks.